### PR TITLE
Update README to reflect tooling changes

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,52 +291,45 @@ Build firmware that can be flashed onto your ESP32 device. The firmware is gener
...
 
 ``` sh
-make esp32
+make esp32 esptool
 ```
 
 If you want to flash the generated firmware on your device, you can use the `firmware`
 too. Internally, the `firmware` tool calls out to `esptool.py` so you need to build
-that first using `make esptool`. Assuming your device is connected through `/dev/ttyUSB0`
+that first using 'make esptool'. Assuming your device is connected through `/dev/ttyUSB0`
```
Comment (kasperl) [inline README.md:299]:
```suggestion
that first using `make esptool`. Assuming your device is connected through `/dev/ttyUSB0`
```

**Comment by kasperl** [comment]
TBR'ed.

File: README.md [hunk 2]
```diff
@@ -351,7 +344,7 @@ The listing produces JSON output that can be processed by other tools:
 ```
 { "hello": {
     "kind" : "snapshot",
-    "id"   : "f0b7e859-9188-52d9-8be3-856bd0e75919"
+    "id"   : "f0b7e859-9188-52d9-8be3-856bd0e75919",
```
Comment (floitsch) [inline README.md:347]: That's not valid JSON.

Comment (kasperl) [reply README.md:347]: Good point.

**Pull request merged**